buzzmobile/process/bearing/bearing.py

```diff
@@ -31,7 +31,16 @@ def median(self):
 bearings['med_filter'] = MedianFilter(rospy.get_param('median_filter_size'))
 bearing_pub = rospy.Publisher('bearing', Float64, queue_size=1)
 
+MIN_FIX_DISTANCE = rospy.get_param('min_fix_distance')
+EARTH_RADIUS = 6.3710088e6
+
+
 def bearing(fix):
+    """
+    Determines if the latest position is >= MIN_FIX_DISTANCE from the last
+    position, and if so adds the new bearing to the median filter and
+    publishes the new median.
+    """
     global last_fix
     global med_filter
```

Comment:
Idk how these two lines passed code review. Can you please remove these `global` lines?